testsuite/test_misc.py

```diff
@@ -23,7 +23,7 @@ def test_export_companies(client):
         f.close()
     wb = load_workbook(fn)
     assert len(wb.worksheets) == 1
-    rows = wb.worksheets[0].rows
+    rows = list(wb.worksheets[0].rows)
```

Comment:
Why cast to list?

Reply:
I updated the `openpyxl` library and now `rows` is a generator, so in order for `len` to be used we need to convert it to a list first (otherwise we'd get a `TypeError: object of type 'generator' has no len()`).